rust/sdk/Cargo.toml

```diff
@@ -12,78 +12,86 @@ license = "Apache-2.0"
 keywords = ["distributed", "decentralized", "event-sourcing"]
 categories = ["network-programming"]
 
-[package.metadata.docs.rs]
-features = ["client"]
-
 [features]
-default = ["client"]
-client = ["reqwest", "url", "bytes", "rand"]
-sqlite = ["rusqlite"]
-arb = ["quickcheck", "rand"]
-with-tokio = ["tokio"]
+with_tokio = ["tokio", "tokio-util", "rand"]
```

Comment:
since we depend on reqwest, we always depend on tokio, so the feature should be removed